src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,21 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (storageDao.get(user.getLogin()).equals(null)) {
+            throw new RuntimeException("Login is exist");
```

Comment:
Using `.equals(null)` to check for null is incorrect; use `==` instead. Also, the exception message should be more informative. According to the checklist, avoid using `RuntimeException`. Consider creating a custom exception like `RegistrationException`.